operators/api/v1alpha2/template_types.go

```diff
@@ -122,6 +122,9 @@ type Environment struct {
 
 	// Name of the storage class to be used for the persistent volume (when needed)
 	StorageClassName string `json:"storageClassName,omitempty"`
+
+	// Whether the instance has to have the user path
+	MountPathEnabled bool `json:"mountPathEnabled,omitempty"`
```

Comment:
Would not set it as `omitempty`, also would add the kubebuilder flag for defaulting it to `true`

Reply:
Added flag for default value and generated the manifests